### DISCUSSION TITLE
Following the build instructions for Pre-built SDK engine results in unknown target

### DISCUSSION THREAD

**Discussion by Zaya6** [o3de/o3de#6132] [Help Wanted] [answered]
Trying to build from source and following the instructions in the documentation here https://o3de.org/docs/welcome-guide/setup/setup-from-github/building-linux/ for the Pre-built SDK engine.

When I run the command 
`cmake --build build/linux --target INSTALL --config profile -j 8`

All I get is 
`ninja: error: unknown target 'INSTALL'
`

**Comment by Zaya6** [comment, marked as answer]
It was just a buggy git clone it seems. I ended up just deleting it and cloning again and now the commands work

**Reply by amznestebanpapp** [reply]
It may also be a case sensitivity thing, the target in ninja is "install"
